package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "step33-2020",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googleinterns/step33-2020.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/googleinterns/step33-2020/issues"
+  },
+  "homepage": "https://github.com/googleinterns/step33-2020#readme",
+  "devDependencies": {
+    "jest": "^26.1.0"
+  },
+  "dependencies": {
+    "babel-jest": "^26.1.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2"
+  },
+  "babel": {
+    "plugins": ["transform-es2015-modules-commonjs"]
```

Comment:
Where do we use babel in this code?

Reply:
We don't directly, but jest is using it under the hood.  Jest isn't set up to directly understand es6 modules, but this babel plugin converts the es6 style modules into commonjs style modules, which jest is then able to understand.